clients/osv/src/funTest/kotlin/OsvServiceFunTest.kt

```diff
@@ -90,6 +90,7 @@ class OsvServiceFunTest : StringSpec({
                 listOf(
                     "GHSA-462w-v97r-4m45",
                     "GHSA-8r7q-cvjq-x353",
+                    "GHSA-fqh9-2qgg-h84h",
```

Comment:
To my understanding, the minimal diff to fix the test would only update the lines 318-417, and line 93. All other updates IIUC are not necessary.

E.g. the modified date is not compared, and the test should be agnostic of ordering of properties.

Reply:
> All other updates IIUC are not necessary.

Might be, but as it's actually quite inconvenient to find out what exactly fails for this test, I've simply serialized the current actual result, pretty-printed it, used it as the new expected result, and then inspected the Git diff afterwards for plausibility.